core/src/main/java/cucumber/runtime/formatter/TestSourcesModel.java

```diff
@@ -0,0 +1,181 @@
+package cucumber.runtime.formatter;
+
+import gherkin.AstBuilder;
+import gherkin.Parser;
+import gherkin.ParserException;
+import gherkin.TokenMatcher;
+import gherkin.ast.Background;
+import gherkin.ast.Examples;
+import gherkin.ast.Feature;
+import gherkin.ast.GherkinDocument;
+import gherkin.ast.Node;
+import gherkin.ast.ScenarioDefinition;
+import gherkin.ast.ScenarioOutline;
+import gherkin.ast.Step;
+import gherkin.ast.TableRow;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class TestSourcesModel {
```

Comment:
I don't think TestSourcesModel is part of the public API but I can't tell for sure. Whole thing should probably be package private.